src/integrationTest/java/uk/gov/hmcts/darts/event/controller/DarNotifyControllerTest.java

```diff
@@ -56,14 +68,50 @@ void setup() {
 
     @Test
     void shouldSendDarNotifyEventSoapAction() throws Exception {
-        darPcStub.respondWithSuccessResponse();
+        darPcStub.respondWithSuccessResponse(OffsetDateTime.now(clock));
+
+        mockMvc.perform(post("/events/dar-notify")
+                            .contentType(APPLICATION_JSON_VALUE)
+                            .content(VALID_NOTIFICATION_JSON))
+            .andExpect(status().is2xxSuccessful());
+
+        darPcStub.verifyNotificationReceivedWithBody(EXPECTED_DAR_PC_NOTIFICATION);
+    }
+
+    @Test
+    @ExtendWith(OutputCaptureExtension.class)
+    void shouldSendDarNotifyEventSoapActionDarPcDateOutSideDriftLimitsRangeBehind(CapturedOutput capturedOutput) throws Exception {
+        OffsetDateTime responseDateTime = OffsetDateTime.now(clock).minusSeconds(90).truncatedTo(ChronoUnit.SECONDS);
+        darPcStub.respondWithSuccessResponse(responseDateTime);
 
         mockMvc.perform(post("/events/dar-notify")
                             .contentType(APPLICATION_JSON_VALUE)
                             .content(VALID_NOTIFICATION_JSON))
             .andExpect(status().is2xxSuccessful());
 
         darPcStub.verifyNotificationReceivedWithBody(EXPECTED_DAR_PC_NOTIFICATION);
+        assertThat(capturedOutput)
+            .containsPattern("Response time from DAR PC is outside max drift limits of PT1M30S. DAR PC Response time: "
```

Comment:
Real minor one but would it be better converting PT1M30S to seconds in the log statement so that it is more human readable?